### PR TITLE
Pr/avatar

### PR DESCRIPTION
**Pull request by h1z3y3**

### CODE REVIEW

**Comment by asim** [comment]
Base64 encoded images need to start with `data:image/png;base64,` or `data:image/png;base64,` so that can be used like:

```
<img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAALQAAAC0CAYAAAA9zQYyAAAABGdBTUEAALGPC/xhBQAAAAlwSFlzA..." />
```

**Comment by asim** [comment]
It might also make sense to upload the avatar to the image service

**Comment by h1z3y3** [comment]
I hadn't known there was a service can upload to CDN before.

I add the feature of uploading CDN, please review ;P

File: avatar/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+Generate an avatar
+
+# Avatar Service
+
+This is the Avatar service
```
Comment (asim) [inline avatar/README.md:5]: Remove the rest of the below readme content and put

```
Generate a unique avatar for your user profiles, NFT crypto punks and much more.
```

**Pull request merged**